core/logger.py

```diff
@@ -1,28 +1,42 @@
 from appdirs import user_log_dir
 import os
-import logging
+import logging.handlers
 
 
 log_directory_name = "irida_uploader"
-log_file_name = 'irida-uploader.log'
+if os.environ.get('IRIDA_UPLOADER_TEST'):
+        log_directory_name = "irida_uploader_test"
```

Comment:
Add a comment here please to say what's going on.  Otherwise things look good!

Reply:
Added comments explaining each line